src/compiler/diagnosticMessages.json

```diff
@@ -1304,7 +1304,7 @@
         "category": "Error",
         "code": 2379
     },
-    "'get' and 'set' accessor must have the same type.": {
+    "'get' and 'set' accessor must have the same type '{0}'.": {

```

Comment:
I'm thinking 

```
Accessors must have identical types, but this 'get' accessor has type '{0}' and its respective 'set' accessor has type '{1}'.
```

with

```
The respective 'set' accessor is declared here.
```

However, to be honest, I'm not sure how I feel about this change; I feel like the simpler error message was actually better.

Reply:
why not just the `the `set` accessor is declared here:`